lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +8,31 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ */
+export interface ISailplaneAPIGatewayProxyEvent extends APIGatewayProxyEvent {
+    /**
+     * HTTP Request body, parsed from a JSON string into an object.
+     * @property body
+     */
+    body: any;
```

Comment:
In speaking with a colleague, I remember that this transformation _only happens_ when the received body has JSON content. So the body can be left untouched, as `string | null`. Since `any` includes `string`, please just add the `| null` which will be true if no body was received.